docs/docs/widgets/checkbox.md

```diff
@@ -1,129 +1,133 @@
 ---
-id: checkbox
-title: Checkbox
----
+
```

Comment:
Hey, you are not supposed to edit this section, can you please revert the changes from this section?